diesel/src/expression/expression_methods/sort_expression_methods.rs

```diff
@@ -0,0 +1,44 @@
+use expression::predicates::*;
+
+#[cfg(feature = "postgres")]
+pub trait Ordering : Sized {
```

Comment:
`SqlOrd` is just a marker trait for SQL types which have a defined order, meaning that `max` and `min` should be called. It shouldn't affect this feature. A trait for this is fine, but I'd prefer `OrderingMethods` or something similar that matches the naming convention of everything else here. (I know it's redundant with the module name, but these traits are exported in prelude so I prefer to have them use non-generic names. It's the same reason that everything in `query_dsl` has `Dsl` in the name.)